cmd/av/stack_sync.go

```diff
@@ -251,6 +264,10 @@ base branch.
 }
 
 func init() {
+	stackSyncCmd.Flags().BoolVar(
+		&syncFlags.All, "all", false,
+		"synchronize all branches",
+	)
```

Comment:
Can you add a mutually exclusive flag with `--current`? I recently learned this is a thing within Cobra: https://github.com/aviator-co/av/pull/133/files#diff-148b0d72c2cfeab9cfe0849ff12194c92c9636b23de06402645f2455a9a6d653R222.

Reply:
TIL. Done.